src/tools/run-make-support/src/external_deps/rustc.rs

```diff
@@ -253,6 +253,13 @@ impl Rustc {
         self
     }
 
+    /// Specify the target CPU.
+    pub fn target_cpu<S: AsRef<str>>(&mut self, target_cpu: S) -> &mut Self {
```

Comment:
Cc @jieyouxu for rmake changes (this seems uncontroversial)

Reply:
Oh, rustbot did that for me 🙃

Reply:
Yeah that is fine